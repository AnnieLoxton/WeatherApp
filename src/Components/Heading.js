import React from 'react';
import '../App.css';


function Heading(){
    return(
        <div className='header'>
        <h1 style={{marginTop: "80px"}}>Weather Forecast</h1>
        <p className='pLarge' style={{marginTop: "16px"}}>
          Search a city to see the current weather.
        </p>
      </div>
    );
}

export default Heading