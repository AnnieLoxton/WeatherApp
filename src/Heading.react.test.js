import React from 'react';
import renderer from 'react-test-renderer';
import Heading from './Components/Heading';

test('renders correctly', () => {
    const tree = renderer.create(<Heading />).toJSON();
    expect(tree).toMatchSnapshot();
    });