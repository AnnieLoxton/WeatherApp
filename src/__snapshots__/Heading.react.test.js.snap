// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <h1
    style={
      Object {
        "marginTop": "80px",
      }
    }
  >
    Weather Forecast
  </h1>
  <p
    className="pLarge"
    style={
      Object {
        "marginTop": "16px",
      }
    }
  >
    Search a city to see the current weather.
  </p>
</div>
`;
